packs/caddy/metadata.hcl

```
@@ -0,0 +1,11 @@
+app {
+  url = "https://caddyserver.com/v2"
+  author = "github.com/mr-karan"
```

hcl formatting; the author is also expected to be that of the app author rather than the pack.
```suggestion
  url    = "https://caddyserver.com/v2"
  author = "Stack Holdings"
```